src/app/content/content.component.html

```diff
@@ -1,24 +1,9 @@
-<div class="content">
-    <mat-tab-group>
-        <mat-tab label="Today's Sales">
-            <mat-grid-list cols="5" rowHeight="3:2">
```

Comment:
I don't think this grid should be removed from the content component for now since its displaying multiple items(Or you can create an **Items-list component** instead)
The Item-component should represent only one item